### DISCUSSION TITLE
Can it generate type from PostgreSQL Json field?

### DISCUSSION THREAD

**Discussion by tombohub** [dotansimha/graphql-code-generator#8231] [Q&A] [answered]
Last time I've tried to generate types from Hasura schema it didn't generate type from JSON field in PostgreSQL database.
Did I miss something or it really cannot generate type from JSON field?

**Comment by ardatan** [comment, marked as answer]
If it doesn't have a type definition with fields in the GraphQL Schema, we cannot generate types from a generic `JSON` scalar.
